style.css

```diff
@@ -503,3 +523,316 @@ ul.points {
 .line2 {
   margin-left: 110px;
 }
+
+.vecto {
+  display: none;
+}
+
+.Imgo {
+  display: none;
+}
+
+.imageo {
+  display: none;
+}
+
+.linee {
+  display: none;
+}
+
+.contact-form-ftext {
+  display: none;
+}
+
+.contact-form-etext {
+  height: 32px;
+  width: 300px;
+  padding: 5px 10px 5px 10px;
+  border: none;
+  border-bottom: 1px solid gainsboro;
+  font-family: 'Inter', sans-serif;
+  margin-left: 24px;
+  margin-top: 20px;
+  font-size: 15px;
+}
+
+.a4 {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .main-header {
+    width: 100%;
+    height: 80px;
+  }
+
+  .header {
+    margin-top: 7px;
+    margin-left: 80px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .nam {
+    display: flex;
+    width: 200px;
+  }
+
+  .men {
+    margin-left: 500px;
+    margin-top: 23px;
+    display: flex;
+    gap: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+  }
+
+  .mail {
+    margin-top: -10px;
+  }
+
+  .section1 {
+    width: 100%;
+    display: flex;
+    height: 870px;
+  }
+
+  .containe2 {
+    display: none;
+  }
+
+  .im {
+    display: flex;
+  }
```

Comment:
- [ ] Your design isn't responsive on all desktop screens. Kindly fix this to make it responsive on all desktop screens 🙏 

![image](https://user-images.githubusercontent.com/74814780/206458992-50cab00b-09e4-4c7d-904b-27e0578a3346.png)